scripts/utils/hardcoded_tokens.json

```diff
@@ -0,0 +1,15 @@
+{
+  "1": {
+    "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359": {
+      "name": "Sai Stablecoin v1.0",
+      "symbol": "SAI",
+      "decimals": 18
+    },
+    "0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2": {
```

Comment:
I checked both entries on etherscan. They are correct

Reply:
Thanks for checking!